homeassistant/components/intellifire/__init__.py

```diff
@@ -0,0 +1,85 @@
+"""The Intellifire integration."""
+from __future__ import annotations
+
+from datetime import timedelta
+import logging
+import logging.handlers
+
+from async_timeout import timeout
+from intellifire4py import IntellifireAsync
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+PLATFORMS: list[str] = [Platform.SENSOR, Platform.BINARY_SENSOR]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Intellifire from a config entry."""
+
+    _LOGGER.info("Setting up config entry: %s", entry.unique_id)
```

Comment:
You should probably drop this to `debug`. HA will log any errors that occur.

Reply:
✅️ - updating this now.